lib/schema_generator/notification_schema_generator.rb

```diff
@@ -48,6 +48,7 @@ def required
         title
         update_type
         user_journey_document_supertype
+        user_need_document_supertype
```

Comment:
Is it OK to make this a required field even though we haven't added it to the publishing API yet? I'd have thought we'd need to make it optional first.